src/Sarif/CodeGenHints.json

```diff
@@ -326,6 +326,21 @@
       }
     }
   ],
+  "Rule.Configuration": [
+    {
+      "kind": "EnumHint",
+      "arguments": {
+        "typeName": "RuleConfiguration",
+        "description": "Values specifying the configuration of a rule.",
+        "memberNames": [
+          "Unknown",
+          "Enabled",
+          "Disabled"
+        ],
+        "allowMemberCountMismatch": true
```

Comment:
You don't need `allowMemberCountMismatch`. There are the same number of `memberNames` here are there are `enum` values in the schema.